command/utils.go

```diff
@@ -14,11 +18,68 @@ func (kw *stdoutkw) Write(k string) (err error) {
 	return err
 }
 
-func HandleClientError(err error) error {
-	if aerr, ok := err.(payload.APIError); ok {
-		if len(aerr.APIError.Fields) > 0 && aerr.Response.StatusCode == 403 {
-			return fmt.Errorf("Validation error: %v", aerr.APIError.Fields)
+func HandleClientError(err *client.APIError, verbose bool) error {
+	if perr, ok := err.Err.(*payload.Error); ok {
+		var errString string
+		// create error message according to response code
+		switch err.Response.StatusCode {
```

Comment:
Can err.Response be nil realistically? then this might result in a panic while handling errors